src/main/webapp/WEB-INF/view/StylizedTextParser.java

```diff
@@ -0,0 +1,80 @@
+// StylizedTextParser.java
+// Class with methods to parse a message with markdown elements and
+// convert to HTML stylized text 
+
+//package codeu.controller;
+
+import java.lang.StringBuffer; 
+import java.lang.String; 
+import java.util.Stack; 
+import java.util.Hashtable; 
+
+public class StylizedTextParser {
+
+    /* Maps markdown symbol to array of HTML opening and closing stylized syntax */
+    private Hashtable<Character, String> styleTable; 
```

Comment:
This should be static and const, and can be named MarkdownHTMLMapping or something like that